bindings/wasm/bindings.js

```diff
@@ -59,6 +83,113 @@ Module.setup = function() {
     return vec[0];
   }
 
+  // CrossSection methods
+
+  const CrossSectionCtor = Module.CrossSection;
+
+  function cross(polygons, fillrule = 'Positive') {
+    if (polygons instanceof CrossSectionCtor) {
+      return polygons;
+    } else {
+      const polygonsVec = polygons2vec(polygons);
+      const cs = new CrossSectionCtor(polygonsVec, fillrule = fillrule);
+      disposePolygons(polygonsVec);
+      return cs;
+    }
+  };
+
+  Module.CrossSection.prototype.translate = function(...vec) {
+    return this._Translate(vararg2vec2(vec));
+  };
+
+  Module.CrossSection.prototype.rotate = function(vec) {
+    return this._Rotate(...vec);
+  };
+
+  Module.CrossSection.prototype.scale = function(vec) {
+    if (typeof vec == 'number') {
+      return this._Scale({x: vec, y: vec});
```

Comment:
Doesn't `vararg2vec2` already take care of this case?

Reply:
Ah perhaps, I just copied what `Manifold.prototype.scale` is doing.

Reply:
Never a bad time to clean up :)